server/clearbit.js

```diff
@@ -477,8 +521,15 @@ export default class Clearbit {
 
       promises.push(asUser.traits(all_traits.user));
 
-      if (domain) {
-        const asAccount = asUser.account({ domain });
+      if (domain || !_.isEmpty(account)) {
+        const accountIdent = _.pick(account, "id", "external_id", "domain");
+
+        // Will need to decide under what conditions we want to do this, but for now do it in all cases
+        if (domain) {
```

Comment:
I do not agree with this; the entire problem we were trying to solve is relinking based on this.

Reply:
yes, the existing behavior is certainly suspect.  I only added the logic to pass back id and external id as well, so that the account remains stable if it's already attached to an account